discovery_engine_core/bindings/cbindgen.toml

```diff
@@ -5,3 +5,27 @@ autogen_warning = "/* Warning, this file is autogenerated by cbindgen. Don't mod
 include_version = true
 sys_includes = ["stdint.h"]
 no_includes = true
+
+header = """
```

Comment:
We don't really need this on generated files.

Reply:
The `header` is mostly there to add `typedef struct A A;` later one (which forces `ffigen` to treat it as an opaque type).

I can remove the license.

Reply:
No.

Alias will generate an alias i.e. `typedef A B;` but we need something which acts as an opaque type definition. (note the missing `struct` part).